src/dropdown.ts

```diff
@@ -1512,69 +1510,65 @@ export default class Dropdown extends LitElement implements FormControl {
       this.isShowSingleSelectIcon = Boolean(
         this.lastSelectedAndEnabledOption?.value,
       );
-    }
 
-    // Dropdown becomes filterable if there are more than 10 options. But input field
-    // won't have rendered yet given we just set `this.isFilterable` above. So we
-    // piggyback off of `this.requestUpdate()` and then wait for the update to complete
-    // before setting the `value` of input field.
-    await this.updateComplete;
```

Comment:
Moved this inside this `else` condition (for single-select) because it only applies to single-select.